packages/amplication-client/src/Plugins/InstalledPluginSettings.tsx

```diff
@@ -114,16 +128,54 @@ const InstalledPluginSettings: React.FC<Props> = ({
             <PluginLogo plugin={plugin} />
             <div className={`${moduleClass}__name`}>{plugin.name}</div>
           </div>
-          <div className={`${moduleClass}__row`}>
+          <div className={`${moduleClass}__column`}>
             <span className={`${moduleClass}__description`}>
               {plugin.description}
             </span>
+            <div className={`${moduleClass}__row`}>
+              <div className={`${moduleClass}__label-title`}>
+                <Label text="Plugin Version" />
+              </div>
+              <SelectMenu
+                title={pluginInstallation.PluginInstallation.version}
+                buttonStyle={EnumButtonStyle.Secondary}
+                className={`${moduleClass}__menu`}
+                icon="chevron_down"
+              >
+                <SelectMenuModal>
+                  <SelectMenuList>
+                    <>
+                      {plugin.versions.map((pluginVersion) => (
+                        <SelectMenuItem
+                          closeAfterSelectionChange
+                          itemData={pluginVersion}
+                          selected={
+                            pluginVersion.version ===
+                            pluginInstallation.PluginInstallation.version
+                          }
+                          key={pluginVersion.id}
+                          onSelectionChange={(pluginVersion) => {
+                            handlePluginInstalledChange(
```

Comment:
version change should not be saved automatically - first show the new settings object and the save button will save all changes including the version selection